### ISSUE TITLE
quoting for datasource_password

### ISSUE THREAD

**Issue by ikcalB** [treydock/puppet-module-keycloak#231]
in `config.cli`, datasource_password is not quoted (like datasource url `connection-url`, i.e.)

please properly quote passwords, otherwise no special characters are able to be used.
(workaround: manually quote passwords in puppet. i.e.: 
``` puppet
...
# TODO: fix manual quoting: "" required inside of ''
datasource_password   => '"some;)special$characters|"'
...
```

**Comment by treydock** [comment]
This will be released as v7.12.1 once Github Actions complete.

**Comment by ikcalB** [comment]
thanks for the quick fix. tested working (y)
